Projects/UOContent/Gumps/Go/GoGump.cs

```diff
@@ -32,6 +32,16 @@ public class GoGump : Gump
 
         private readonly LocationTree m_Tree;
 
+        static GoGump()
+        {
+            Felucca = new LocationTree("felucca", Map.Felucca);
```

Comment:
We you can assign it to the field directly and eliminate the static constructor.

Reply:
You want to say write `GoGump()` instead of `static GoGump()`?

Reply:
The way it was before?
```
if (from.Map == Map.Ilshenar)
            {
                tree = Ilshenar ??= new LocationTree("ilshenar", Map.Ilshenar);
            }
```